open_academy/views/menus.xml

```diff
@@ -0,0 +1,6 @@
+<odoo>
+    <menuitem id="menu_root" name="Courses"></menuitem>
```

Comment:
follow this documentation to name the id of your record views https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action